### PR TITLE
Adding simple Azure Worker Role Sample app

### PR DESCRIPTION
**Pull request by miker1423**

### CODE REVIEW

File: Samples/2.0/AzureWorkerRoleSample/OrleansClient/OrleansClient.csproj [hunk 1]
```diff
@@ -0,0 +1,262 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ReubenBond) [inline Samples/2.0/AzureWorkerRoleSample/OrleansClient/OrleansClient.csproj]: Could this be converted to the new project format?

Comment (miker1423) [reply Samples/2.0/AzureWorkerRoleSample/OrleansClient/OrleansClient.csproj]: I tried creating a new project and adding the necessary packages, but I wasn't able to add it as a Worker Role...

File: Samples/2.0/AzureWorkerRoleSample/Silo/WorkerRole.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            var proxyPort = RoleEnvironment.CurrentRoleInstance.InstanceEndpoints["OrleansProxyEndpoint"].IPEndpoint.Port;
+            var siloEndpoint = RoleEnvironment.CurrentRoleInstance.InstanceEndpoints["OrleansSiloEndpoint"].IPEndpoint;
+            var connectionString = RoleEnvironment.GetConfigurationSettingValue("DataConnectionString");
+            var deploymentId = RoleEnvironment.DeploymentId;
+
+            var builder = new SiloHostBuilder()
+                .Configure<ClusterOptions>(options =>
+                {
+                    options.ClusterId = deploymentId;
+                    options.ServiceId = deploymentId;
```
Comment (ReubenBond) [inline Samples/2.0/AzureWorkerRoleSample/Silo/WorkerRole.cs]: The ServiceId here should be a fixed value, like "AzureWorkerRoleSample"

**Comment by miker1423** [comment]
Thanks to both of you! 😄 and congrats for the 2.0 release! 🎉

**Pull request merged**